docs/gen-docs/kyma_alpha_enable_module.md

```diff
@@ -0,0 +1,56 @@
+---
+title: kyma alpha enable module
+---
+
+Enables a module in the cluster or in the given Kyma resource.
+
+## Synopsis
+
+Use this command to enable Kyma modules available in the cluster.
+
+### Detailed description
+
+For more information on Kyma modules, see the 'create module' command.
```

Comment:
I am not sure, what you expect here as a `link`? This is quite common to tell the user which command he should call, a link to the implementation would not be helpful in my opinion

Reply:
@jeremyharisch I assumed- presumably incorrectly - that you used the phrase `For more information (...), see (...)` to direct the reader to the source of information on the topic. If not, maybe the sentence should be rephrased. I'll leave it for you to decide, though.

Reply:
IMO this is fine. Since this is the output which will be printed out to the user if he runs the help command on `kyma alpha enable module`, thus this redirects the user direclty to run `kyma alpha create module` for more information.

Meaning, this is not a document where the user will look at, it is just pure CLI output

Reply:
Now I get it 😄 Thanks for the explanation.